docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-16-04.md

```diff
@@ -2,241 +2,174 @@
 author:
   name: Linode
   email: docs@linode.com
-description: 'This guide will teach you basic setup and configuration of Linux, Nginx, MySQL and PHP on Ubuntu 16.04'
-keywords: ["nginx", "lemp", "php", "ubuntu 16.04"]
+description: 'This guide will teach you basic setup and configuration of Linux, NGINX, MySQL, and PHP on Ubuntu.'
```

Comment:
Probably should have a longer description if you want to maximize Google search visibility